src/site/paragraphs/table.drupal.liquid

```diff
@@ -1,57 +1,54 @@
-{% comment %}
-    Example data:
-    entity: {
-        "entityId": "3944",
-        "fieldTable": {
-            "tableValue": "Row 1 Column 1 Row 1 Column 2 Row 1 Column 3 Row 2 Column 1 Row 2 Column 2 Row 2 Column 3 Row 3 Column 1 Row 3 Column 2 Row 3 Column 3",
-            "value": {
-                "0": [
-                "Row 1 Column 1",
-                "Row 1 Column 2",
-                "Row 1 Column 3"
-                ],
-                "1": [
-                "Row 2 Column 1",
-                "Row 2 Column 2",
-                "Row 2 Column 3"
-                ],
-                "2": [
-                "Row 3 Column 1",
-                "Row 3 Column 2",
-                "Row 3 Column 3"
-                ],
-            }
-            "format": "rich_text",
-            "caption": "Test Table"
-        }
-    }
-{% endcomment %}
+<!-- Mobile -->
+<div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-display--none"
```

Comment:
You can probably remove this surrounding div and move its properties onto the table element [here](https://github.com/department-of-veterans-affairs/vets-website/pull/14746/files#diff-52295a65c2fc05a75586bce4505377d1ff1ef84394b519485aec840743f39a29R5)